docs/release-information/update-to-version-3-200/update-platform-and-modules-to-3-200.md

```diff
@@ -39,41 +40,42 @@ See the [details](https://github.com/VirtoCommerce/vc-build#update).
+| 24 | [VC Cart](https://github.com/VirtoCommerce/vc-module-cart) | [3.200](https://github.com/VirtoCommerce/vc-module-cart/releases/tag/3.200.0) |
+| 25 | [VC Inventory](https://github.com/VirtoCommerce/vc-module-inventory) | [3.200](https://github.com/VirtoCommerce/vc-module-inventory/releases/tag/3.200.0) |
+| 26 | [VC Customer](https://github.com/VirtoCommerce/vc-module-customer) | [3.200](https://github.com/VirtoCommerce/vc-module-customer/releases/tag/3.200.0) |
+| 27 | [VC Orders](https://github.com/VirtoCommerce/vc-module-order) | [3.200](https://github.com/VirtoCommerce/vc-module-order/releases/tag/3.200.0) |
+| 28 | [VC File sistem assets](https://github.com/VirtoCommerce/vc-module-filesystem-assets) | [3.200](https://github.com/VirtoCommerce/vc-module-filesystem-assets/releases/tag/3.200.0) |
+| 29 | [VC Azure blob assets](https://github.com/VirtoCommerce/vc-module-azureblob-assets) | [3.200](https://github.com/VirtoCommerce/vc-module-azureblob-assets/releases/tag/3.200.0) |
+| 30 | [VC Marketing](https://github.com/VirtoCommerce/vc-module-marketing) | [3.200](https://github.com/VirtoCommerce/vc-module-marketing/releases/tag/3.200.0) |
+| 31 | [VC Content](https://github.com/VirtoCommerce/vc-module-content) | [3.200](https://github.com/VirtoCommerce/vc-module-content/releases/tag/3.200.0) |
+| 32 | [VC Tax](https://github.com/VirtoCommerce/vc-module-tax) | [3.200](https://github.com/VirtoCommerce/vc-module-tax/releases/tag/3.200.0) |
+| 33 | [VC XApi](https://github.com/VirtoCommerce/vc-module-experience-api) | [3.200](https://github.com/VirtoCommerce/vc-module-experience-api/releases/tag/3.200.0) |
+| 34 | [VC WebHooks](https://github.com/VirtoCommerce/vc-module-webhooks) | [3.200](https://github.com/VirtoCommerce/vc-module-webhooks/releases/tag/3.200.0) |
+| 35 | [VC Subscription](https://github.com/VirtoCommerce/vc-module-subscription) | [3.200](https://github.com/VirtoCommerce/vc-module-subscription/releases/tag/3.200.0) |
+| 36 | [VC Sitemaps](https://github.com/VirtoCommerce/vc-module-sitemaps) | [3.200](https://github.com/VirtoCommerce/vc-module-sitemaps/releases/tag/3.200.0) |
+| 37 | [VC Bulk actions](https://github.com/VirtoCommerce/vc-module-bulk-actions) | [3.200](https://github.com/VirtoCommerce/vc-module-bulk-actions/releases/tag/3.200.0) |
 
-## FAQ
+## FAQs
 
 ### How can I migrate my extension module?
-1. You need to install all required environment, like VS 2022 and .NET 6 SDK
-2. Bump all dependent VirtoCommerce NuGet packages. (At least to 3.200 version)
-3. Fix all issues and gain the solution compile successful.
-4. Note, during developing and debugging you should not compile your module either with .NET Core 3.1 and .NET 6 at the same time.
+1. Make sure all required prerequisites, such as VS 2022 and .NET 6 SDK, have been installed to your environment.
+2. Bump all dependent VirtoCommerce NuGet packages to at least version 3.200.
```

Comment:
I saw FluentValidation at least was updated to 10.x version to prevent BadImageException. @akak1977 Is there any other dependencies which was updated due to migration to .NET 6?